src/manifests/test_manifest.py

```diff
@@ -37,8 +37,7 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
                 - with-security
                 - without-security
             smoke-test:
-              test-spec:
-                - spec.yaml
+              test-spec: spec.yaml
```

Comment:
Can we keep it consistent here?
`.yaml` or `.yml`?

Reply:
Sure updated to `.yml`.